iib/workers/tasks/build.py

```diff
@@ -158,34 +158,41 @@ def _finish_request_post_build(
         index image.
     :param str overwrite_from_index_token: the token used for overwriting the input
         ``from_index`` image.
+    :return: pull spec of the index_image
+    :rtype: str
     :raises IIBError: if the manifest list couldn't be created and pushed
     """
     conf = get_worker_config()
+    state_reason = ''
     if from_index and overwrite_from_index:
         log.info(f'Ovewriting the index image {from_index} with {output_pull_spec}')
         index_image = from_index
         exc_msg = f'Failed to overwrite the input from_index container image of {index_image}'
         args = [f'docker://{output_pull_spec}', f'docker://{index_image}']
         _skopeo_copy(*args, copy_all=True, dest_token=overwrite_from_index_token, exc_msg=exc_msg)
+        state_reason = f'Overwrote the index image {from_index} with {output_pull_spec}'
```

Comment:
Could you reuse this as the log message like you did in the `elif` branch?

Reply:
yeah, the reason I added again here was that I was trying to keep consistencies with the past tenses for state_reason, used in this function. Let me change it to use log message